### DISCUSSION TITLE
First version number

### DISCUSSION THREAD

**Discussion by HarelM** [HaylLtd/cordova-plugin-background-geolocation#14] [General]
The question here is which version to start from. Either a completely new plugin i.e `1.0` or continue the numbering of the old one - i.e `4.x`.
I'm in favor of starting over - 1.0 for the first version.

Side note:
Just wanted to split this discussion to its right place.

**Comment by RaddishIoW** [comment]
Yep - I've been flip-flopping on this question over the last couple of days, but I think that as we're changing the package name, it makes sense to restart the version numbers.

So, the "interim" version, which will maintain compatibility with the current version of @mauron85/cordova-plugin-background-goelocation will be version 1.0.
The "next major version" should probably be version 2.0 - as this will have potentially breaking changes, like AndroidX.
